eva/optimizer/rules/rules.py

```diff
@@ -610,6 +612,25 @@ def apply(self, before: LogicalUnion, context: OptimizerContext):
         return after
 
 
+class LogicalGroupByToPhysical(Rule):
+    def __init__(self):
+        pattern = Pattern(OperatorType.LOGICALGROUPBY)
+        pattern.append_child(Pattern(OperatorType.DUMMY))
+        super().__init__(RuleType.LOGICAL_GROUPBY_TO_PHYSICAL, pattern)
+
+    def promise(self):
+        return Promise.LOGICAL_GROUPBY_TO_PHYSICAL
+
+    def check(self, before: Operator, context: OptimizerContext):
+        return True
+
+    def apply(self, before: LogicalGroupBy, context: OptimizerContext):
```

Comment:
unclear what is happening. A brief explanation would be nice regarding the intent would be nice

Reply:
Consistent with other optimizer rules.